src/Symfony/Component/Validator/Constraints/File.php

```diff
@@ -93,17 +93,17 @@ private function normalizeBinaryFormat($maxSize)
         if (ctype_digit((string) $maxSize)) {
             $this->maxSize = $sizeInt;
```

Comment:
This looks like to be the only use of `$sizeInt` in this function. You should inline the L91 cast.